### PR TITLE
Fr 51 create business rules function

### PR DESCRIPTION
**Pull request by VladimirKarin**
https://vladimirkarin.atlassian.net/browse/FR-51

### CODE REVIEW

**Comment by fedosejev** [comment]
Typo in the branch name: `rulles`

File: businessRules/users.js [hunk 1]
```diff
@@ -34,15 +34,88 @@ function login(req, res) {
...
+    res.status(200).json({
+        status: 'OK',
+        statusMessage: 'You have successfully logged out.',
+    });
+}
+
+function updateUser(userId, key, value) {
+    let users = getUsers();
+
+    const user = users.find((user) => userId === user.id);
```
Comment (fedosejev) [inline businessRules/users.js:64]: What will happen if user is not found?

File: index.js [hunk 2]
```diff
@@ -36,6 +36,15 @@ app.post('/login', (req, res) => {
     login(req, res);
 });
 
+app.get('/login', (req, res) => {
```
Comment (fedosejev) [inline index.js:39]: Is it an API endpoint for users to log in?

Comment (VladimirKarin) [reply index.js:39]: I've changed it to 
res.status(200).json(login(req, res));
And it worked in Postman. I've did something wrong?

**Review by fedosejev — APPROVED** [review]

**Pull request merged**